use-cases/using-SRIOV-and-kata.md

```diff
@@ -0,0 +1,313 @@
+01:00.0 Ethernet controller: Intel Corporation Ethernet Controller 10-Gigabit X540-AT2 (rev 03)
+
+...
+$ #sudo required below to read the card capabilities
+
+$ sudo lspci -s 01:00.0 -v
+01:00.0 Ethernet controller: Intel Corporation Ethernet Controller 10-Gigabit X540-AT2 (rev 03)
+        DeviceName: NIC1
+        Subsystem: Dell Ethernet 10G 4P X540/I350 rNDC
+        Flags: bus master, fast devsel, latency 0, IRQ 146
+        Memory at 91c00000 (64-bit, prefetchable) [size=2M]
+        Memory at 91e04000 (64-bit, prefetchable) [size=16K]
+        Expansion ROM at 92800000 [disabled] [size=512K]
+        Capabilities: [40] Power Management version 3
+        Capabilities: [50] MSI: Enable- Count=1/1 Maskable+ 64bit+
+        Capabilities: [70] MSI-X: Enable+ Count=64 Masked-
+        Capabilities: [a0] Express Endpoint, MSI 00
+        Capabilities: [e0] Vital Product Data
+        Capabilities: [100] Advanced Error Reporting
+        Capabilities: [150] Alternative Routing-ID Interpretation (ARI)
+        Capabilities: [160] Single Root I/O Virtualization (SR-IOV)
+        Capabilities: [1d0] Access Control Services
+        Kernel driver in use: ixgbe
+        Kernel modules: ixgbe
+```
+
+The important line being:
+
+```
+        Capabilities: [160] Single Root I/O Virtualization (SR-IOV)
+```
+
+If your card does not report this capability, then it does not support SR-IOV.
+
+### IOMMU Groups and PCIe Access Control Services
+
+Run the following command to see how the IOMMU groups are setup on your 
+host system. 
+```
+$ find /sys/kernel/iommu_groups/ -type l
+```
+
+The command's output details whether or not your NIC is setup 
+appropriately with respect to PCIe Access Control Services (ACS). If the PCI 
+bridge is within the same IOMMU group as your NIC, it indicates 
+that either your device does not support ACS or your device does not 
+appropriately share this default capability.
+
+If the IOMMU groups are setup properly, you will have the PCI for each ACS-enabled 
+NIC port in its own IOMMU group. If you do not see any output when running the above
+command, then you likely need to update your host's kernel configuration.
+
+For more details, see http://vfio.blogspot.com/2014/08/iommu-groups-inside-and-out.html
+
+### Update the host kernel
+
+
+Depending on your host kernel configuration, you might have to rebuild the 
+kernel. If the following conditions are true, you do not need to rebuild 
+your kernel: 
+
+- `CONFIG_VFIO_IOMMU_TYPE1`, `CONFIG_VFIO`, and `CONFIG_VFIO_PCI` are set in 
+the kernel configuration. Your kernel is built with VFIO support when 
+configurations are set. 
+- `CONFIG_VFIO_NOIOMMU` is disabled in the host kernel configuration.
+
+See the following steps one through three if you need to rebuild the kernel.
+
+The following steps, which are based on the Ubuntu 16.04 distribution, update 
+the SR-IOV host system. If you use a different distribution, make 
+appropriate adjustments to the commands:
```

Comment:
Can we note they may also wish to update the kernel version numbers referenced in the commands to 'the latest' - and maybe hint/show how to find the latest version as well?